cookietemple/config/config.py

```diff
@@ -107,6 +107,20 @@ def config_pat() -> None:
             access_token = cookietemple_questionary_or_dot_cookietemple(
                 "password", "Please enter your Github Access token"
             )
+            # validate input token
+            if not access_token.startswith("ghp_") or access_token.endswith(" "):  # type: ignore
```

Comment:
Think this is overcomplicated. I would:

1. print the warning if it does not start with "ghp_"
2. Always strip the token, because it is an instant operation anyways

Reply:
> Always strip the token, because it is an instant operation anyways

I mean, it does not matter in any ways (performance wise), but it's just an unnecessary operation if the token is already valid.

Reply:
Yeah, but your code is more complex. (Still very simple, but more complex). I would handle it straight forward.
If the token is valid the check would also be unnecessary :)